code_snippets: tarantool_config_storage setup fix (#5571)
tarantool/doc: doc/code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage/README.md

```console
$ tt start tarantool_config_storage
```

> To upload configuration for [config_storage](../config_storage) application, execute `put_config` function for the leader instanse (instance with RW mode):

Reviewer: ```suggestion
To upload configuration for the [config_storage](../config_storage) application, execute the `put_config()` function for the leader instance (the instance in the RW mode):
```
